src/core/config/platform.h

```diff
@@ -146,4 +146,14 @@
 #endif
 #endif
 
+/**
+ * @def OPENTHREAD_CSL_UNCERT
+ *
+ * The Uncertainty of the scheduling CSL of transmission by the parent, in ±10 us units.
+ *
+ */
+#ifndef OPENTHREAD_CSL_UNCERT
+#define OPENTHREAD_CSL_UNCERT UINT8_MAX
```

Comment:
Consider keeping the convention for macros naming in this file: `OPENTHREAD_CONFIG_PLATFORM_`

Reply:
done